dll/win32/browseui/shellbars/CBandSiteMenu.cpp

```diff
@@ -386,6 +386,21 @@ HRESULT STDMETHODCALLTYPE CBandSiteMenu::QueryContextMenu(
     return MAKE_HRESULT(SEVERITY_SUCCESS, 0, USHORT(idMax - idCmdFirst +1)); 
 }
 
+HRESULT CBandSiteMenu::_ShowToolbarError(HRESULT hRet)
+{
+    WCHAR szText[260];
+    WCHAR szTitle[256];
+
+    if (!LoadStringW(GetModuleHandleW(L"browseui.dll"), IDS_TOOLBAR_ERR_TEXT, szText, _countof(szText)))
+        wcscpy(szText, L"Cannot create toolbar.");
```

Comment:
Please don't use this function. You want String(Cb/Cch)Copy(Ex)W or wcscpy_s.

Reply:
Done, thanks!